piqe_ocp_lib/api/resources/ocp_pods.py

```diff
@@ -183,3 +183,20 @@ def get_pod_node(self, namespace, pod_name):
             if api_response.spec["nodeName"]:
                 node_name = api_response.spec["nodeName"]
         return node_name
+
+    def list_of_pods_in_a_node(self, node_name, namespace=None, label_selector=None):
```

Comment:
Can you please add typing to this method definition. I realized the rest of the methods don't have it but we have started retroactively adding typing to all the methods ever since we switched to python 3.6+

Reply:
@btjd 

I tried to get the response without namespace and only with label_selector as "kubevirt.io"
```
In [2]: import os
   ...: os.environ["KUBECONFIG"]="/opt/src/ftf/kubeconfig_cnv"
   ...: from piqe_ocp_lib.api.resources import OcpPods
   ...: pod_obj = OcpPods()
   ...: pod_obj.list_of_pods_in_a_node("worker-0-1","kubevirt.io")
Out[2]: ['virt-launcher-rhel8-brown-flyingfish-ntbtw', 'virt-handler-7sggp']
```
but this outputs 'virt-handler-7sggp' pod as well which is not be the intended result.

Hence I believe we should have the namespace argument as well.